ng-dev/utils/git/github.ts

```diff
@@ -8,7 +8,6 @@
 
 import type {OctokitOptions} from '@octokit/core/dist-types/types.js';
 
-import {graphql} from '@octokit/graphql';
```

Comment:
Can we drop the `BUILD.bazel` dependency and extra npm install maybe?

Also: We expose some class from `@octokit/graphql` for error handling (in some other util file). Do we know this will still work, given it being potentially a different instance? (e.g. `instanceof` may no longer work)